tests/framework/PostProcessors/FTimporterPostProcessor/gold/FTimporter_and/PrintPS.xml

```diff
@@ -0,0 +1 @@
+<data name="PrintPS" type="Pointset"><input>BE2,BE3,BE1</input><output>TOP</output><inputFilename>PrintPS.csv</inputFilename></data>
```

Comment:
you really need all these XML files????This comment is valid for all the tests files you added